app/(site)/social-support/socialAidData.tsx

```diff
@@ -0,0 +1,20 @@
+import { Feature } from "@/types/feature";
+import { AlertCircle, HelpCircle } from "lucide-react"; // Example icons from a library
+
+const socialAidData: Feature[] = [
+  {
+    id: 1,
+    icon: <AlertCircle />, // Icon for the first feature (you can use any icon library)
+    title: "Fundo de Apoio Extraordinário de Emergência Social (FAEES)",
```

Comment:
Are we waiting on a form for this section? I remember we talked about this being something that students could apply for, but I'm not aware of how they can do so